.readthedocs.yml

```
@@ -1,8 +1,24 @@
+version: 2
+
+build:
+  image: latest
+
+sphinx:
+  builder: html
+  configuration: docs/conf.py
+  fail_on_warning: true
+
+# Install regular dependencies.
+# Then, install special pinning for RTD.
 python:
-  version: 3.5
-  pip_install: true
+  version: 3.8
+  system_packages: false
+  install:
+    - method: pip
+      path: .
+      extra_requirements:
+        - docs
+        - all
```

I see that you have not switched to declaring your dependencies in `setup.cfg` yet, so maybe try this?

```suggestion
    - requirements: docs/requirements.txt
```